app/pages/lab/workflows.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { Link } from 'react-router';
+import DragReorderable from 'drag-reorderable';
+import ModalFormDialog from 'modal-form/dialog';
+import Paginator from '../../talk/lib/paginator';
+import WorkflowCreateForm from './workflow-create-form';
+import LoadingIndicator from '../../components/loading-indicator';
+
+const WorkflowsPage = (props) => {
+  const meta = props.workflows.length ? props.workflows[0].getMeta() : {};
+
+  const renderWorkflow = ((workflow) => {
+    return (
+      <li key={workflow.id}>
+        <Link key={workflow.id} to={props.labPath(`/workflow/${workflow.id}`)} className="nav-list-item" activeClassName="active">
+          {workflow.display_name}
+          {workflow.id === props.project.configuration.default_workflow && (
+            <span title="Default workflow">{' '}*{' '}</span>
```

Comment:
Let's add an explanation at the top for the asterisk now we have the real estate for it.